crates/bevy_gizmos/src/config.rs

```diff
@@ -135,6 +146,9 @@ pub struct GizmoConfig {
     ///
     /// Gizmos will only be rendered to cameras with intersecting layers.
     pub render_layers: RenderLayers,
+
+    /// Describe how lines should join
+    pub line_joints: Option<GizmoLineJoint>,
```

Comment:
It feels clearer to just have a `GizmoLineJoint::None` or `Sharp` and avoid the option here.